python/mathsys/dev/ir/code.py

```
@@ -0,0 +1,168 @@
+    def limit(self, limit: parser.Limit) -> u32:
+        return self.new(bytes(ir.Limit(
+            variable = self.variable(limit.variable),
+            approach = self.expression(limit.approach),
+            direction = u8(int(limit.direction) + 1) if limit.direction is not None else null8(),
+            nest = self.nest(limit.nest),
+            exponent = self.expression(limit.exponent) if limit.exponent is not None else null32()
+        )))
+    #~ IR -> 5 LEVEL GENERATION
+    def level5(self, level5: parser.Level5) -> u32:
+        match level5:
+            case parser.Infinite(): return self.infinite(level5)
```

Ruff (0.14.6)

Multiple statements on one line (colon)

(E701)